--- openrc-run.sh
+++ openrc-run-new.sh
@@ -257,31 +257,31 @@
 	export EINFO_VERBOSE
 fi
 
-for _cmd; do
-	if [ "$_cmd" != status -a "$_cmd" != describe ]; then
-		# Apply any ulimit defined
-		[ -n "${rc_ulimit:-$RC_ULIMIT}" ] && \
-			ulimit ${rc_ulimit:-$RC_ULIMIT}
-		# Apply cgroups settings if defined
-		if [ "$(command -v cgroup_add_service)" = "cgroup_add_service" ]
-		then
-			if grep -qs /sys/fs/cgroup /proc/1/mountinfo
-			then
-				if [ -d /sys/fs/cgroup -a ! -w /sys/fs/cgroup ]; then
-					eerror "No permission to apply cgroup settings"
-					break
-				fi
-			fi
-			cgroup_add_service
-		fi
-		[ "$(command -v cgroup_set_limits)" = "cgroup_set_limits" ] &&
-			cgroup_set_limits
-		[ "$(command -v cgroup2_set_limits)" = "cgroup2_set_limits" ] &&
-			[ "$_cmd" = start ] &&
-			cgroup2_set_limits
-		break
-	fi
-done
+#for _cmd; do
+#	if [ "$_cmd" != status -a "$_cmd" != describe ]; then
+#		# Apply any ulimit defined
+#		[ -n "${rc_ulimit:-$RC_ULIMIT}" ] && \
+#			ulimit ${rc_ulimit:-$RC_ULIMIT}
+#		# Apply cgroups settings if defined
+#		if [ "$(command -v cgroup_add_service)" = "cgroup_add_service" ]
+#		then
+#			if grep -qs /sys/fs/cgroup /proc/1/mountinfo
+#			then
+#				if [ -d /sys/fs/cgroup -a ! -w /sys/fs/cgroup ]; then
+#					eerror "No permission to apply cgroup settings"
+#					break
+#				fi
+#			fi
+#			cgroup_add_service
+#		fi
+#		[ "$(command -v cgroup_set_limits)" = "cgroup_set_limits" ] &&
+#			cgroup_set_limits
+#		[ "$(command -v cgroup2_set_limits)" = "cgroup2_set_limits" ] &&
+#			[ "$_cmd" = start ] &&
+#			cgroup2_set_limits
+#		break
+#	fi
+#done
 
 eval "printf '%s\n' $required_dirs" | while read _d; do
 	if [ -n "$_d" ] && [ ! -d "$_d" ]; then
